code/src/index.css

```diff
@@ -5,9 +5,162 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: black;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+#root {
+  margin: 0;
+}
+.album-wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  width: 100%;
+  justify-content: center;
+  gap: 2rem;
+  margin: auto;
+  align-items: center;
+}
+
+.album {
+  flex-direction: column;
+  max-width: 300px;
+}
+.artist-name a {
+  font-size: 14px;
+  color: #a0a0a0;
+  text-decoration: transparent;
+  font-family: Helvetica, sans-serif;
+}
+
+.artist-name:hover {
+  text-decoration-color: #a0a0a0;
+}
+
+.artist-name {
+  margin: 5px 0px;
+  font-size: 14px;
+  color: #a0a0a0;
+  text-decoration: transparent;
+  font-family: Helvetica, sans-serif;
+}
+
+.album-title {
+  margin: 5px 0px;
+}
+.album-title a {
+  font-size: 14px;
+  color: #ffffff;
+  text-decoration: none;
+  font-family: Helvetica, sans-serif;
+}
+
+header {
+  color: white;
+  margin: 10px 0px;
+}
+h1 {
+  padding-left: 60px;
+}
+.icons {
+  position: absolute;
+  inset: 0;
```

Comment:
this one is really good to know! No more top, left, right, bottom-styling from now on :)